ase/scripts/afu_sim_setup

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+# Copyright(c) 2013-2017, Intel Corporation
+#
+# Redistribution  and  use  in source  and  binary  forms,  with  or  without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of  source code  must retain the  above copyright notice,
+#   this list of conditions and the following disclaimer.
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+# * Neither the name  of Intel Corporation  nor the names of its contributors
+#   may be used to  endorse or promote  products derived  from this  software
+#   without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,  BUT NOT LIMITED TO,  THE
+# IMPLIED WARRANTIES OF  MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED.  IN NO EVENT  SHALL THE COPYRIGHT OWNER  OR CONTRIBUTORS BE
+# LIABLE  FOR  ANY  DIRECT,  INDIRECT,  INCIDENTAL,  SPECIAL,  EXEMPLARY,  OR
+# CONSEQUENTIAL  DAMAGES  (INCLUDING,  BUT  NOT LIMITED  TO,  PROCUREMENT  OF
+# SUBSTITUTE GOODS OR SERVICES;  LOSS OF USE,  DATA, OR PROFITS;  OR BUSINESS
+# INTERRUPTION)  HOWEVER CAUSED  AND ON ANY THEORY  OF LIABILITY,  WHETHER IN
+# CONTRACT,  STRICT LIABILITY,  OR TORT  (INCLUDING NEGLIGENCE  OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+import sys
+import os
+import errno
+import shutil
+from os.path import dirname, realpath, sep
+import subprocess
+
+
+#
+# Figure out the root of the ASE source directory.  ASE is installed
+# along with OPAE SDK, so either find it in the installation tree or in the
+# source tree.
+#
+def getASESrcPath():
+    # CMake will update any variables marked by @ with the proper values.
+    opae_install_dir = '@CMAKE_INSTALL_PREFIX@'
+    project_src_dir = '@CMAKE_CURRENT_SOURCE_DIR@'
```

Comment:
Will this script work when installed from an RPM?

Reply:
Good question, though not unique to this change since other scripts are using the same method. I will investigate and fix all of them, if necessary, as a separate commit.

Reply:
I meant separate pull, in case that wasn't clear. I'd like to pull this one now and fix the general problem next, if it exists.